src/panels/config/cloud/account/cloud-account.js

```diff
@@ -182,7 +190,7 @@ class CloudAccount extends EventsMixin(LocalizeMixin(PolymerElement)) {
 
   _formatSubscription(subInfo) {
     if (subInfo === null) {
-      return "Fetching subscription…";
+      return "[[localize('ui.panel.config.cloud.account.fetching_subscription')]]";
```

Comment:
This is not in inside the template function of Polymer, so we can't use Polymer bindings and it should not be a string:
```suggestion
      return this.hass.localize('ui.panel.config.cloud.account.fetching_subscription');
```